platforms/macos/src/node.rs

```diff
@@ -523,6 +523,11 @@ declare_class!(
                 .unwrap_or(false)
         }
 
+        #[method(isAccessibilityEnabled)]
+        fn enabled(&self) -> bool {
```

Comment:
Also, `is_enabled` would be more appropriate. Please rename this method and update below.